tests/runtime/usdt

```diff
@@ -192,33 +192,33 @@ REQUIRES bash -c "exit 1"
 
 NAME usdt probes - attach to probe with probe builtin and args by file
 PROG usdt:./testprogs/usdt_test:* { printf("%lld %s\n", arg0, probe ); exit(); }
-EXPECT usdt:./testprogs/usdt_test:tracetest:testprobe
+EXPECT_REGEX usdt:./testprogs/usdt_test:tracetest:testprobe
 TIMEOUT 5
 BEFORE ./testprogs/usdt_test
 REQUIRES ./testprogs/systemtap_sys_sdt_check
 
 NAME usdt probes - attach to probe with probe builtin and args by file with child
 RUN {{BPFTRACE}} -e 'usdt:./testprogs/usdt_test:* { printf("%lld %s\n", arg0, probe ); exit(); }' -c ./testprogs/usdt_test
-EXPECT usdt:./testprogs/usdt_test:tracetest:testprobe
+EXPECT_REGEX usdt:./testprogs/usdt_test:tracetest:testprobe
 TIMEOUT 5
 
 NAME usdt probes - attach to probe with probe builtin and args by pid
 RUN {{BPFTRACE}} -e 'usdt:./testprogs/usdt_test:* { printf("%lld %s\n", arg0, probe ); exit(); }' -p {{BEFORE_PID}}
-EXPECT usdt_test:tracetest:testprobe
+EXPECT_REGEX usdt_test:tracetest:testprobe
 TIMEOUT 5
 BEFORE ./testprogs/usdt_test
 REQUIRES ./testprogs/systemtap_sys_sdt_check
 
 NAME usdt probes - attach to probe with semaphore
 RUN {{BPFTRACE}} -e 'usdt::tracetest:testprobe { printf("%s\n", str(arg1) ); exit(); }' -p {{BEFORE_PID}}
-EXPECT tracetest_testprobe_semaphore: [1-9]
+EXPECT_REGEX tracetest_testprobe_semaphore: 1
```

Comment:
Why this has `EXPECT_REGEX` but the similar tests below have `EXPECT`?
(I'm not very familiar with the USDT semaphores)